packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -100,6 +101,7 @@ Future<void> buildWindows(WindowsProject windowsProject, BuildInfo buildInfo, {
     globals.printStatus(
       'A summary of your Windows bundle analysis can be found at: ${outputFile.path}',
     );
+    CodeSizeEvent('windows').send();
```

Comment:
ya, I might be missing something but at a glance, it seems like everything you need is already in analyzeAotSnapshot.

Reply:
Done!